virttest/remote_commander/remote_runner.py

```diff
@@ -295,6 +297,10 @@ def parse_func_name(self, func_name, commander):
         :param func_name: Function name
         :param commander: Where to execute the command (remote or local)
         """
+        for name in func_name:
```

Comment:
As far as I know, yes. Once the code is accepted, it's supported, but I believe you're close. But this should not break anything so I guess the second ack will not be as hard to get, right guys? ...

PS: Would you please add the `NotImplemented` fix as currently it crashes when you use `commander.async.python_file_run_with_helper`, with the fix it raises exception, unless you use `commander.manage.python_file_run_with_helper`.